src/platform/testing/e2e/nightwatch-commands/openUrl.js

```diff
@@ -0,0 +1,14 @@
+exports.command = function openUrl(url, disableForesee = true) {
+  this.url(url);
+
+  if (disableForesee) {
+    this.execute(() => {
+      const css = document.createElement('style');
+      css.type = 'text/css';
+      css.innerHTML = '.__acs { display: none !important; }';
```

Comment:
Foresee's API has a few methods of suppressing the modal, but none of them worked reliably for me. This does, though.

Reply:
Clever.